### PR TITLE
sdk: Add NAME to credentials service

### LINKED ISSUE #569: add new name field in api for creds
**Is this a BUG REPORT or FEATURE REQUEST?**:
bug report

**What happened**:
we need an ability to manage creds based on name. currently creds are being managed using a uuid, which can be inconvenient. so we need a new constant to access name.

**What you expected to happen**:
Add new const for name around here:
https://github.com/libopenstorage/openstorage/blob/master/api/api.go#L72

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Container Orchestrator and version:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by lpabon**
**What this PR does / why we need it**:
Add `name` to credentials as it is done for Volumes and Snapshots

**Which issue(s) this PR fixes** (optional)
Closes #569

**Special notes for your reviewer**:

* Had to remove the tests for the sdk from Travis-ci since they would fail as sdk-test would need to be updated
* SDK version bumped to 0.2.0

### CODE REVIEW

File: api/api.proto [hunk 1]
```diff
@@ -1379,13 +1381,14 @@ message SdkSchedulePolicyInterval {
   // Number of instances to retain
   int64 retain = 1;
 
+  // Start oneof at field number 200 to allow for expansion
   oneof period_type {
     // Daily policy
-    SdkSchedulePolicyIntervalDaily daily = 2;
+    SdkSchedulePolicyIntervalDaily daily = 200;
```
Comment (sdeoras) [inline api/api.proto]: how are these related to cred name change?

Comment (lpabon) [reply api/api.proto]: Due to this change, I wanted to prepare the message for future ones that may come. I'm making sure we have enough field numbers to allow for expansion while trying to keep the oneof's at the end of the message. That way we can add more in the oneof's.

Long story short, this change brought up that we needed to make sure that our messages were easily enhanced. All others messages are ok, just the ones with oneof needed to be adjusted.
